### PR TITLE
P#60597 Remove Default Form from installation and the fix for including autoload file

### PR DESCRIPTION
**Pull request by tung-le-esg**
P#60597 Remove Default Form from installation and the fix for including autoload file

### CODE REVIEW

File: plugin/Installer/DatabaseChanges.php [hunk 1]
```diff
@@ -281,39 +281,6 @@ private function getCreateQueryForms()
...
-				'template' => $template,
-				'country_active' => 1,
-				'zip_active' => 1,
-				'street_active' => 1,
-				'radius_active' => 1,
-				'geo_order' => 'street,zip,city,country,radius'
-			)
-		);
-		$defaultFormId = $this->_pWPDB->insert_id;
-		$this->installDataQueryFormFieldConfig($defaultFormId);
```
Comment (jayay) [inline plugin/Installer/DatabaseChanges.php:313]: How about this method (`installDataQueryFormFieldConfig`)? Isn't this unused as well?

**Review by jayay — COMMENTED** [review]
Please see my comment. I wonder how this can lead to a -0.7% code coverage decrease 😃

File: plugin/Installer/DatabaseChanges.php (hunk 1 above)
Comment (tung-le-esg) [reply plugin/Installer/DatabaseChanges.php:313]: Yep. I removed this function.

**Comment by tung-le-esg** [comment]
> Please see my comment. I wonder how this can lead to a -0.7% code coverage decrease smiley

I re-run the action and the code coverage is increased by +0.4%.

File: plugin/Installer/DatabaseChanges.php (hunk 1 above)
Comment (jayay) [reply plugin/Installer/DatabaseChanges.php:313]: 👍🏻

**Review by jayay — APPROVED** [review]

**Comment by michaelarnold-dev** [comment]
@tung-le-esg Could you please increase the code coverage?

**Comment by tang-hien-egs** [comment]
> @tung-le-esg Could you please increase the code coverage?

@michaelarnold-dev , i fixed it, can you check PR ?

**Review by jayay — APPROVED** [review]

**Pull request merged**